homeassistant/components/smartthings/sensor.py

```diff
@@ -53,371 +56,391 @@ class SmartThingsSensorEntityDescription(SensorEntityDescription):
-                key=Attribute.volume,
+                key=Attribute.VOLUME,
                 name="Volume",
                 native_unit_of_measurement=PERCENTAGE,
             )
         ]
     },
-    Capability.battery: {
-        Attribute.battery: [
+    Capability.BATTERY: {
+        Attribute.BATTERY: [
             SmartThingsSensorEntityDescription(
-                key=Attribute.battery,
+                key=Attribute.BATTERY,
                 name="Battery",
                 native_unit_of_measurement=PERCENTAGE,
                 device_class=SensorDeviceClass.BATTERY,
                 entity_category=EntityCategory.DIAGNOSTIC,
             )
         ]
     },
-    Capability.body_mass_index_measurement: {
-        Attribute.bmi_measurement: [
+    # no fixtures
+    Capability.BODY_MASS_INDEX_MEASUREMENT: {
+        Attribute.BMI_MEASUREMENT: [
             SmartThingsSensorEntityDescription(
-                key=Attribute.bmi_measurement,
+                key=Attribute.BMI_MEASUREMENT,
                 name="Body Mass Index",
                 native_unit_of_measurement=f"{UnitOfMass.KILOGRAMS}/{UnitOfArea.SQUARE_METERS}",
                 state_class=SensorStateClass.MEASUREMENT,
             )
         ]
     },
-    Capability.body_weight_measurement: {
-        Attribute.body_weight_measurement: [
+    # no fixtures
+    Capability.BODY_WEIGHT_MEASUREMENT: {
+        Attribute.BODY_WEIGHT_MEASUREMENT: [
             SmartThingsSensorEntityDescription(
-                key=Attribute.body_weight_measurement,
+                key=Attribute.BODY_WEIGHT_MEASUREMENT,
                 name="Body Weight",
                 native_unit_of_measurement=UnitOfMass.KILOGRAMS,
                 device_class=SensorDeviceClass.WEIGHT,
                 state_class=SensorStateClass.MEASUREMENT,
             )
         ]
     },
-    Capability.carbon_dioxide_measurement: {
-        Attribute.carbon_dioxide: [
+    # no fixtures
+    Capability.CARBON_DIOXIDE_MEASUREMENT: {
+        Attribute.CARBON_DIOXIDE: [
             SmartThingsSensorEntityDescription(
-                key=Attribute.carbon_dioxide,
+                key=Attribute.CARBON_DIOXIDE,
                 name="Carbon Dioxide",
                 native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
                 device_class=SensorDeviceClass.CO2,
                 state_class=SensorStateClass.MEASUREMENT,
             )
         ]
     },
-    Capability.carbon_monoxide_detector: {
-        Attribute.carbon_monoxide: [
+    # no fixtures
+    Capability.CARBON_MONOXIDE_DETECTOR: {
+        Attribute.CARBON_MONOXIDE: [
             SmartThingsSensorEntityDescription(
-                key=Attribute.carbon_monoxide,
+                key=Attribute.CARBON_MONOXIDE,
                 name="Carbon Monoxide Detector",
             )
         ]
     },
-    Capability.carbon_monoxide_measurement: {
-        Attribute.carbon_monoxide_level: [
+    # no fixtures
```

Comment:
Please remove all similar comments in a follow up PR as these are test comment in the production code